spec/product_spec.rb

```diff
@@ -2,5 +2,64 @@
 require_relative "../models/product"
 
 describe Product do
-  it "belongs to a class Product"
+  let(:product){Product.new("MacBook Pro", 1200)}
+
+  it "belongs to a class Product" do
+    expect(product).to be_a(Product)
+  end
+  it "has a String for a title" do
+    expect(product.title).to be_a(String)
+  end
+  it "has contains a price that is an Integer" do
+    expect(product.cost).to eq(1200)
+  end
+  it "contains a list of reviews" do
+    expect(product.reviews).to be_a(Array)
+  end
+
+  describe "#read_product_info" do
+    context "when we read over our product's info" do
+      it "should print out a string with it's title and cost" do
+        expect(product.read_product_info).to eq("#{product.title} currently costs #{product.cost}")
```

Comment:
could also separate these out: `expect(product.title).to eq("Macbook")` ... etc..